src/turing_model.jl

```diff
@@ -0,0 +1,10 @@
+# TODO: create the turing_model function:
```

Comment:
I personally like writing this kind of stuff in issues more. It's more satisfying to close an issue than to remove a TODO comment xD